Release/src/http/client/http_client_asio.cpp

```diff
@@ -25,12 +25,19 @@
 #pragma clang diagnostic ignored "-Wunused-local-typedef"
 #pragma clang diagnostic ignored "-Winfinite-recursion"
 #endif
+
 #include <boost/algorithm/string.hpp>
+#include <boost/asio/steady_timer.hpp>
+#include <boost/bind.hpp>
+
+#if defined(CPPREST_BOTAN_SSL)
+#include <botan/asio_stream.h>
+#else
```

Comment:
Please comment else and endif:

#if defined(CPPREST_BOTAN_SSL)
#else // ^^^ CPPREST_BOTAN_SSL // !CPPREST_BOTAN_SSL vvv
#endif // CPPREST_BOTAN_SSL

Reply:
Occurs elsewhere.

I know the existing codebase is somewhat bad about this, but that's led to several nasty bugs before. I'd like us to follow that convention for substantial new functionality being added like this.